index.js

```diff
@@ -0,0 +1,58 @@
+/* Rockets
+let list = document.getElementById("blocks").childNodes;
+for (let i = 0; i < list.length; i++) {
+list[i].addEventListener("click",
+function() {
+blocks.insertBefore(this, blocks.childNodes[0])
+});
+}
+*/
+
+// Travelers - work in progress 
```

Comment:
Similarly to what you've done with the Rockets above, you can loop through an array of the `<div>` elements, assign an `addEventListener('mousedown')` and `addEventListener('mouseup')`